### ISSUE TITLE
Menu only mode on Mobile

### ISSUE THREAD

**Issue by ryanerwin** [angusmcleod/discourse-quick-messages#43]
On mobile, QM's composer / reader interface doesn't have as obvious a benefit compared to desktop mode, especially since you can't press CTRL+ENTER on mobile to enter a newline... Each newline results in a sent message with now way to insert a newline.

When we implement the "Menu mode only" feature, maybe we allow the choice of Desktop or Mobile...

Curious to hear what @angusmcleod  thinks about this idea?

**Comment by angusmcleod** [comment]
Yeah I like this idea. I think there is a benefit in having a "Quick Message" style mobile UI, bu why force a forum to use it? Sure, let's do it.
